lib/elastic_record/search_hits.rb

```diff
@@ -1,7 +1,7 @@
 module ElasticRecord
   class SearchHits
     attr_accessor :hits, :model
-    delegate(*Enumerable.instance_methods, to: :hits)
+    delegate(:[], *Enumerable.instance_methods, to: :hits)
```

Comment:
I don't like this. Why are any delegations required?

Reply:
Oh, I can get rid of the `:[]`. A consumer's test was digging into the implementation details of a search hit. Delegating the Enumerable methods make it easier to convert existing code, and I think it makes sense, since it's a wrapper around a collection.

Reply:
What about

```
include Enumerable
delegate :each, to: :hits
```

Reply:
I just dropped it after talking to Matt IRL.